certbot-ci/certbot_integration_tests/.coveragerc

```diff
@@ -0,0 +1,6 @@
+[run]
+disable_warnings = module-not-imported,no-data-collected
```

Comment:
Because Coverage will complain about `acme`, `certbot` and `certbot_nginx` packages not been imported from the thread that executed the coverage for theses packages. Indeed, certbot is called from the command line using a subprocess.

The coverage works correctly however, and theses warnings are harmless messages, coming from a not standard execution, and then can be disabled.

Reply:
Great explanation! Let's stick it in there as a comment so future people know why the code is like that.

Reply:
Done.